package.json

```diff
@@ -68,7 +68,7 @@
   },
   "packageManager": "yarn@1.22.22",
   "engines": {
-    "node": "^16.20 || ^18.16 || >=20",
+    "node": "^18.16 || ^20.14 || >=22",
```

Comment:
ESLint 9 requires at least ^18.18, so can we bump to that to avoid another breaking change later?

Reply:
Sure!